interviews/week2.md

```diff
@@ -0,0 +1,233 @@
+
+**Amazing! I guess that answers that everyone should sightread the pools! ((Please don't)) Next up... Oh yeah! Congrats on the win by the way, though you got into a tiebreaker and from a different perspective, that's a sign of early game struggle. How do you think the strategies you've mentioned for this round will change in the later stages?**
+
+**Frossno**: I think that in later stages, we'll be able to integrate more of our "unga bunga" strat considering that we have four rice powerhouses
+
+**arccat**: THE MONKEY IS REAL
+
+**DiamondGenius75**: unga bunga strat :moyai:
+
+**iid3rp**: it'll change drastically in my perspective; as the opponents goes stronger and we need to change strategies, plans, and map pickings, how we play and who will play those maps…
+
+**Tenshi** — For me, I prefer not getting pressured and focus more on maps that can improve my skill.
+
+**DiamondGenius75**:
+
+![](https://cdn.discordapp.com/attachments/1050992500950708385/1068221713952809030/image.png)
+
+**What kind of changes you think will happen if 2 of the powerhouses was unable to perform on any day? Would you panic or feel hopeless?**
+
+**Frossno**: It's unavoidable to panic, even just a little. The best we can do is adapt.
+
+**iid3rp**: I'd say we give it another shot of hope somehow
+
+**Frossno**: We’d just change our strategy based off of the remaining players
+
+**iid3rp**: there's no such thing as being hopeless on a thing you're very hopeful to win it or get it
+
+**arccat**: That's true.
+
+**Tenshi**: As one of the Rice and SV powerhouses (got to include this bruh), I think the best thing to do is to use Plan B.
+
+**Okay! I think the media centre has run out of bullets- I mean questions. One last thing! You're facing Hong Kong this Ro16 and possibly facing Canada, France, Ireland, or Vietnam by Quarterfinals. Do you have any message for any of these countries?**
+
+**Dyei**: please be gentle
+
+**iid3rp**: choke me daddy AHAHAHHAHAHAH
+
+**arccat**: SUSSY
+
+**Tenshi**: Be weary of what’s going to come. 
+
+**Frossno**: I'm excited to face off against the new Hong Kong! Please remember that we won't be going down without a fight, and that we're in this to win the whole thing. For Canada, France, Ireland, and Vietnam, let's all have fun!
+
+**iid3rp**: I'd say, good luck and we hope that we can make this tournament the most of it because we're glad to even qualify on 4dm at the very first place
+
+**Frossno**: Raiden dies.
+
+**Dyei**: facing these teams is going to be a hard fight but don’t even try to underestimate the PH team, yes early game is probably our weakness, but we are still improving and like Frossno, our team captain said, we are not going down without a fight. (the audacity for me to say "we" but most likely I’m still not going to play because my acc is bad)
+
+**Alright! This sums up the whole interview, thank you so much for answering! Good bye and see you in space!**
+
+**dyei**: Thank you. 
+
+**Frossno**: Take you, Arccat!
+

```

Comment:
oh you forgot to remove these lines, it was reserved for poland but you wrote below thailand (thats fine keep the below one as is and remove these lines)